src/package/components/Tabs/index.js

```diff
@@ -22,13 +22,20 @@ class TabsComponent extends Component {
     return (
       <Tabs onSelect={this.handleSelect}>
 
-        <TabList className='list pa0 ma0 flex flex-wrap br2 br--top'>
+        <TabList
+          className='list pa0 ma0
+                    flex br2 br--top
+                    items-center'
```

Comment:
This is going to break views when the container is small and the tabs need to wrap